lms/djangoapps/shoppingcart/models.py

```diff
@@ -824,6 +824,73 @@ def generate_pdf_invoice(self, course, course_price, quantity, sale_price):
 
         return pdf_buffer
 
+    def __unicode__(self):
+        return "company: {}".format(self.company_name)
```

Comment:
I like the idea of having a unicode representation of this model for debugging purposes, but I think we should include more information in the description.  In addition to the company name, I think the most important identifying information is stored in the `internal_reference` and `customer_reference_number` fields.